docs/examples/cookbook/drag-and-drop.md

```diff
@@ -198,7 +198,7 @@ local function TodoEntry(
 			Name = "TodoEntry",
 
 			Size = UDim2.fromScale(1, 1),
-			BackgroundColor3 = scope:Computed(function(use)
+			BackgroundColor3 = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
			BackgroundColor3 = scope:Computed(function(use)
```

Reply:
reverted